### DISCUSSION TITLE
post/mock-test-code-part-2/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#37] [Announcements]
# post/mock-test-code-part-2/

테스트 코드로부터 받은 피드백으로 구현 코드를 지속적으로 리팩토링하기

https://tech.kakaopay.com/post/mock-test-code-part-2/

**Comment by bellCold** [comment]
좋은 글 감사합니다~~~ :)

**Comment by pak0426** [comment]
재밌게 잘 읽고갑니다!!
